.github/build_real.sh

```diff
@@ -16,6 +16,7 @@ touch  ~/.config/pip/constraints.txt
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$IPOPT_DIR/lib/pkgconfig
 
 pip install .[optview,testing] -v
+pip install cyipopt # for testing IPOPT
```

Comment:
I'm wondering why we need to install cyipopt as we don't install IPOPT itself and we skip all the IPOPT tests anyway?

Reply:
This script is used only by the Docker-based testing, not the GHA steps added in this PR. In #425 I added `cyipopt` to the `testing` set of dependencies for pip, here I moved it out of the testing dep into its own line here, such that we can run the test suite without IPOPT installed and those tests should be skipped correctly.
In the future, we should move cyipopt out of this layer and into the deps layer of Docker, so that the Python wrapper for IPOPT is already there if IPOPT is installed.

Reply:
Right I confused myself with two different setups, thanks!